lib/pages/chat/chat_room_search_mixin.dart

```diff
@@ -49,7 +55,7 @@ mixin ChatRoomSearchMixin {
 
   void initializeSearch({
     required Timeline? Function()? getTimeline,
-    Function(int)? scrollToIndex,
+    Future Function(int)? scrollToIndex,
```

Comment:
Is this equivalent to Future<void> or any Future?

Reply:
it is same

Reply:
Future<void> I wanted to say sorry*